app/controllers/friendships_controller.rb

```diff
@@ -17,7 +21,16 @@ def create
 
 	def destroy
 		@friendship = Friendship.find(params[:id])
+		if @friendship.friend == current_user
```

Comment:
Why did `destroy` action need to have this `if else` block that assigns `@user` ? Couldn't it simply assign the `@friendship` then directly do `@friendship.destroy` ?

Reply:
Because we need a @user var in the friendship request form in views